blockstore/apps/api/v1/views/bundles.py

```diff
@@ -91,3 +108,20 @@ def get_serializer_class(self):
             return BundleVersionWithFileDataSerializer
         # Generic model serializer is sufficient for other views.
         return BundleVersionSerializer
+
+    @action(detail=True)
+    def redirect(self, request, *args, **kwargs):
+        """
+        Redirect a BundleVersion file path to the actual file.
+
+        Check <decisions/0003-bundle-versions-redirect.rst> for more context.
+        """
+        instance = self.get_object()
+        snapshot = instance.snapshot()
+        serialized_snapshot = snapshot.serialize(context={'request': request})
+
+        filename = request.query_params.get('file')
```

Comment:
(Ignore this comment) Why a querystring instead of part of the URL? If it's filename-like, putting it in the URL would look cooler.